<launch>
    <arg name="namespace" default="red"/>
    
    <!-- arg name="depth_topic" default="camera/depth_registered/points"/>
    <arg name="camera_topic" default="camera/color/image_raw/compressed"/ -->

    <arg name="camera_topic" default="red_color_filter/brick_mask/compressed"/>
    <arg name="depth_topic" default="/camera/depth/points"/>
    

    <node name="pointcloud_filter" type="pointcloud_filter" pkg="pointcloud_filter" ns="$(arg namespace)">
        <remap from="/erl_husky/camera/depth_registered/points" to="$(arg depth_topic)"/>
        <remap from="/color_filter/brick_mask/compressed" to="$(arg camera_topic)"/>
        <remap from="/color_filter/nContours" to="red_color_filter/nContours"/>
        <rosparam command="load"  file="$(find pointcloud_filter)/config/brick_distance_parameters.yaml"/>
    </node>
</launch>